src/test/java/org/elasticsearch/plugin/zentity/ModelsActionTest.java

```diff
@@ -8,77 +10,91 @@
 import static org.junit.Assert.fail;
 
 public class ModelsActionTest {
-    private static Exception assertInvalidEntityType(String entityType) {
-        return ModelsAction.validateEntityType(entityType)
-                .map((ex) -> {
-                    assertTrue(ex.getMessage().contains("Invalid entity type [" + entityType + "]"));
-                    return ex;
-                })
-                .orElseGet(() -> {
-                    fail("failure expected");
-                    return null;
-                });
+
+    private static Exception assertInvalidStrictName(String name) {
```

Comment:
Might make sense to move these tests to an `io.zentity.model.ModelTest` class now that the method has moved?

Reply:
+ agreed I'll move those tests over now